### PR TITLE
Use the average perimeter as the center of a polygon or line.

### PR DESCRIPTION
**Pull request by manthey**
When a polygon has many points near each other in one area and few along the opposite part of the perimeter, this produces a nicer center.

### CODE REVIEW

**Comment by manthey** [comment]
@matthewma7 Would this be useful as a stand-alone function in utils?  It could be named something like `utils.centerFromPerimeter(arrayOfPoints)` where the points are objects with `x`, `y` values.

**Comment by matthewma7** [comment]
@manthey Yes, I think this could be useful, at this moment I have a use case for it. 
I think the function signature you suggested looks good.

File: src/util/index.js [hunk 1]
```diff
@@ -745,6 +745,39 @@ var util = module.exports = {
...
 
+  /**
+   * Return the coordinate associated with the center of the perimeter formed
+   * from a list of points.
+   *
+   * @param {geo.geoPosition[]} coor An array of coordinates.
+   * @returns {geo.geoPosition|undefined} The position for the center, or
+   *    `undefined` if no such position exists.
+   */
+  centerFromPerimeter: function (coor) {
```
Comment (aashish24) [inline src/util/index.js]: @manthey it would be good to describe the math in words to explain what is done in this snippet.

Comment (manthey) [reply src/util/index.js]: @aashish24 I added a bit more to the function comment.

Comment (aashish24) [reply src/util/index.js]: thanks, I found it useful.

**Pull request merged**